README.md

```diff
@@ -28,38 +28,42 @@ Example configuration for **Metadefender Cloud** usage:
 [general]
 BaseUrl=https://api.metadefender.com/v4/file
 ConstantUsers=5
-TestDuration=10
+InjectDuration=30
+MaxDuration=60
 ScanWorkflow=multiscan,sanitize
 LocalPath=/home/opswatuser/testfiles
 PollingIntervals=500
 ApiKey=1234567890abcdefghijklmnopqrstuv
-WaitBeforePolling=1000
+ScanNumberUpperBound=13
```

Comment:
Maybe `ScanRequestsUpperBound` would be a better name for this variable.
And please use a default value of 100 here.